examples/java/hello_world/MainActivity.java

```diff
@@ -127,4 +131,19 @@ private void makeRequest() {
         .start(Executors.newSingleThreadExecutor())
         .sendHeaders(requestHeaders, true);
   }
+
+  private void recordStats() {
+    final Counter counter = engine.statsClient().counter(new Element("foo"), new Element("bar"),
+                                                         new Element("counter"));
+
+    final Gauge gauge =
+        engine.statsClient().gauge(new Element("foo"), new Element("bar"), new Element("gauge"));
+
+    counter.increment(1);
+    counter.increment(5);
+
+    gauge.set(5);
+    gauge.add(5);
+    gauge.sub(5);
```

Comment:
Maybe vary these numbers?

Reply:
> Also what ref of Envoy are you bumping to? Doesn't look like the `main` branch?

my local branch is pointed to a different version of envoy, will revert